tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
-# installation
+# Installation
+
+This tutorial describes how to install a Marble node on a server. 
+
+## Prerequisites
+
```

Comment:
Should a section be added for recommended hardware or should we assume the user already knows based on the required software?